_docs/_latest/setup/upgrade.md

```diff
@@ -5,71 +5,44 @@ order: 00
 
 # {{ page.title }}
 
-This guide explains how to use the Forseti upgrade tool.
+This guide explains how to upgrade your Forseti from v1.x to v2.
 
-## Before you begin
+## Important notes
 
-Before you upgrade Forseti, you'll need the following:
+ * We currently don't support data migration from v1 to v2, so you will need to archive the database manually 
+   for future reference if the data is important to you.
+ * [Forseti v2 configuration]({% link _docs/latest/configure/forseti/index.md %}) is different than v1 so 
+   you can not replace the v2 configuration file with the v1 configuration file.
+ * Resources in inventory are no longer configurable.
```

Comment:
Configuration of the resources that are inventoried is not configureable

Reply:
Updated